internal/petstore/handlers/util.go

```diff
@@ -0,0 +1,27 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/anz-bank/sysl-go/common"
+)
+
+//  newString creates a string ptr
+func newString(s string) *string {
+	return &s
+}
+
+// setJSONResponseContentType sets the content type of the response to the appropriate application/json value.
+func setJSONResponseContentType(ctx context.Context) {
+	headers := common.RequestHeaderFromContext(ctx)
+	headers["Content-Type"] = []string{"application/json; charset=utf-8"}
+}
+
+// newRequestContext creates a new request context
```

Comment:
Nit. Can we have fewer than three copies of this code?

```suggestion
//  newString creates a string pointer.
func newString(s string) *string {
	return &s
}

// setJSONResponseContentType sets the content type of the response to the appropriate application/json value.
func setJSONResponseContentType(ctx context.Context) {
	headers := common.RequestHeaderFromContext(ctx)
	headers["Content-Type"] = []string{"application/json; charset=utf-8"}
}

// newRequestContext creates a new request context.
```

Reply:
ok